narwhals/_dask/expr.py

```diff
@@ -490,6 +490,44 @@ class DaskExprStringNamespace:
     def __init__(self, expr: DaskExpr) -> None:
         self._expr = expr
 
+    def replace(
+        self,
+        pattern: str,
+        value: str,
+        *,
+        literal: bool = False,
+        n: int = 1,
+    ) -> DaskExpr:
+        return self._expr._from_call(
+            lambda _input, _pattern, _value, _literal, _n: _input.str.replace(
+                _pattern, _value, n=n, regex=not literal
+            ),
+            "replace",
+            pattern,
+            value,
+            n,
+            literal,
+            returns_scalar=False,
+        )
+
+    def replace_all(
+        self,
+        pattern: str,
+        value: str,
+        *,
+        literal: bool = False,
+    ) -> DaskExpr:
+        return self._expr._from_call(  # pragma: no cover , coverage bug?
```

Comment:
coverage seems not to be hitting this line somehow?

Reply:
It should be because in `narwhals.Expr`, `replace_all` is defined by calling `replace`, hence it doesn't trigger the call to the compliant `replace_all` but to the compliante `replace`

Reply:
aahh, of course!! Should I make it call `replace_all` just for consistency? even though it is just one paramter change.

Reply:
In my opinion yes: I always hope that under the hood there is some optimization going on for (at least some) backends